.github/workflows/update-helm-docs.yaml

```diff
@@ -0,0 +1,38 @@
+name: Update Helm Docs
+
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  update-helm-docs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.23
+
+    - name: Run helm-docs
+      run: |
+        cd .github/workflows
+        chmod +x ./update-helm-docs.sh
+        ./update-helm-docs.sh
+
+    - name: Commit changes
+      run: |
+        git config --local user.email "action@github.com"
```

Comment:
Use the mapcolonies-bot user email (check `shared-workflows` for example)

Reply:
done